sdk/signalr/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/__init__.py

```diff
@@ -0,0 +1,188 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
+__all__ = ["build_authentication_token", "WebPubSubServiceClient"]
+
+from datetime import datetime, timedelta
+from typing import TYPE_CHECKING
+
+import jwt
+
+import azure.core.credentials as corecredentials
+import azure.core.pipeline as corepipeline
+import azure.core.pipeline.policies as corepolicies
+import azure.core.pipeline.transport as coretransport
+
+
+# Temporary location for types that eventually graduate to Azure Core
+from .core import rest as corerest
+from ._version import VERSION as _VERSION
+from ._policies import JwtCredentialPolicy
+from ._utils import UTC as _UTC
+
+if TYPE_CHECKING:
+    from azure.core.pipeline.policies import HTTPPolicy, SansIOHTTPPolicy
+    from typing import Any, List, cast, Type, TypeVar
+    ClientType = TypeVar('ClientType', bound='WebPubSubServiceClient')
+
+
+def build_authentication_token(endpoint, hub, key, **kwargs):
```

Comment:
We also need a `from_connection_string` version of this function.
Do you have any comments on this?